pettingzoo/butterfly/knights_archers_zombies/src/weapons.py

```diff
@@ -8,17 +8,16 @@
 BLACK = (0, 0, 0)
 GRAY = (200, 200, 200)
 ARROW_SPEED = 45
-SPEED_VAL = 3
 ANGLE_RATE = 10
-KNIGHT_SPEED = 5
+SWORD_SPEED = 20
```

Comment:
why is this change needed?

Reply:
For some reason it wasn't used at all (actually a lot of defined variables aren't used at all, seems to be forgotten between updates), so I removed it. And then in line 63 of the old file, the speed was hard defined to be 20, which I changed to be a global variable to be consistent with the rest of the file.